src/hotspot/share/opto/compile.cpp

```diff
@@ -2148,7 +2144,11 @@ void Compile::Optimize() {
     set_for_igvn(save_for_igvn);
   }
 
-  // Perform escape analysis
+  // Now that all inlining is over and no PhaseRemoveUseless will run, cut edge from root to loop
+  // safepoints
+  remove_root_to_sfpts_edges(igvn);
+
+    // Perform escape analysis
```

Comment:
Indent is wrong for this line. Please, fix it.

Reply:
fixed